### PR TITLE
Leaderboard setup

### PR DESCRIPTION
**Pull request by adarachel**
## Project requirements
IMPORTANT NOTE This is the first time you will use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Keep that in mind!
Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise).
Write plain HTML markup with minimum styling (just to make the layout)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "project-leaderboard",
+  "version": "1.0.0",
+  "description": "a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (Shisui6) [inline package.json]: Great job setting up webpack for your project👍. However, you forgot to add a script to start the webpack server for development. You can do this by adding `"start": "webpack serve --open"`  to your scripts in your package.json file.🙏. This way, its easy to startthe webpack server

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @adarachel , 

# Requires Changes 🍀
You have done a great job so far!👏
Your design is good 🖼️, and your code follows best practices. However, there are a few issues that you still need to work on🪲.

## Highlights
- Webpack set up correctly ✅
- Design adhers to wireframe ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (peterrobert) [reply package.json]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @adarachel, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**